modules/bootstrapped/test/src/smithy4s/builder/EndpointBuilderSpec.scala

```diff
@@ -0,0 +1,70 @@
+package smithy4s
```

Comment:
**issue:** this should go to `core/test` - it has a dependency on `bootstrapped` so you should be able to move it pretty much directly

Reply:
actually it doesn't have a dependency and the test is right where it should be. Attempting to add a dependency created a cycle because of SBT's model which led the build to crash.

Reply:
oh. welp, I should've double-checked then :P